faithcomesbyhearing/dbp
# FCBH-1900 Backend Download Storage Filling up

# Description
- Created a downloads folder for the temporary files
- Added `php artisan delete:temporaryZipFiles` command to delete files with more than 1 hour of creation
- Added DeleteTemporaryZipFiles command to the cron jobs, every hour will be executed

## Issue Link
[FCBH-1900](https://fullstacklabs.atlassian.net/browse/FCBH-1900) 

## How Do I QA This
- Run `php artisan delete:temporaryZipFiles` command and verify that all the files older than 1 hour on the `public/downloads` folder are deleted

## Checklist
